specification/compute/resource-manager/readme.md

```diff
@@ -237,6 +237,11 @@ directive:
       - $.definitions.VirtualMachine
     suppress:
       - BodyTopLevelProperties
+  - where:
+      - $.definitions.VirtualMachineScaleSet
+    suppress:
+      - BodyTopLevelProperties
+    reason: Placement (introduced in version 2025-04-01) is an ARM level property
```

Comment:
Sorry, did not get this. what do you mean ?

Reply:
For more context:
https://github.com/Azure/azure-rest-api-specs/pull/31425#issuecomment-2513560567